doc/en/user/source/rest/api/styles.rst

```diff
@@ -94,7 +95,7 @@ Controls a given style.
    * - GET
      - Return style ``s``
      - 200
-     - SLD, HTML, XML, JSON
+     - SLD, HTML, XML, JSON, ZIP
      - HTML
```

Comment:
The SLDPackageHandler.encode manages about this. But maybe is better to just delegate to sldHandler.encode and return the SLD raw file. What do you think?